src/index.html

```diff
@@ -8,6 +8,29 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img src="images/imac.jpeg" alt="product"
```

Comment:
if there are more cards, then each image will have the same alternative text, which does not give the user an understanding of what is in the image